xbmc/addons/PVRClient.cpp

```diff
@@ -785,6 +786,46 @@ PVR_ERROR CPVRClient::IsRecordable(const EPG::CConstEpgInfoTagPtr &tag, bool *is
   return retVal;
 }
 
+bool CPVRClient::IsPlayable(const EPG::CConstEpgInfoTagPtr &tag)
+{
+  if (!m_bReadyToUse)
+    return false;
+
+  try
+  {
+    EPG_TAG pvrTag;
+    WriteEpgTag(tag, pvrTag);
+    return m_pStruct->IsPlayable(pvrTag);
+  }
+  catch (std::exception &e)
+  {
+    LogException(e, __FUNCTION__);
+  }
+
+  return false;
+}
+
+const std::string CPVRClient::GetEpgTagUrl(const EPG::CConstEpgInfoTagPtr &tag)
+{
+  if (!m_bReadyToUse)
+    return NULL;
+
+  try
+  {
+    EPG_TAG pvrTag;
+    WriteEpgTag(tag, pvrTag);
+    char url[4096];
```

Comment:
Its the maximum URL size we might expect. I don't think Kodi handles longer URLs, neither do most browsers. Shall I extract it to a constant or is there already one?